cmd/serviceaccount.go

```diff
@@ -102,6 +103,15 @@ func getValidEntity() (string, string, error) {
 	return entityType, id, nil
 }
 
+func getValidRole() (string, error) {
+	var fullRole string
+	if len(workspaceId) > 0 {
+
+		fullRole = "WORKSPACE_ADMIN"
```

Comment:
@andriisoldatenko to match Orbit, I think we want to use `WORKSPACE_VIEWER` as the default role, if the entity type is `WORKSPACE` and `DEPLOYMENT_VIEWER` if the entity type is `DEPLOYMENT`.

Reply:
@schnie 
Ashb suggested to have flag `--role=admin` and generate `WORKSPACE_ADMIN `
if `--role=viewer`  generate WORKSPACE_VIEWER? right?

Reply:
That works for me

Reply:
So we should essentially concatenate `entityType` and `role`?

Reply:
yes

Reply:
if user skip flag role, should it be viewer as default value?

Reply:
yep